mmv1/products/networksecurity/ServerTlsPolicy.yaml

```diff
@@ -20,6 +20,8 @@ update_verb: :PATCH
 update_mask: true
 description: |
   ClientTlsPolicy is a resource that specifies how a client should authenticate connections to backends of a service. This resource itself does not affect configuration unless it is attached to a backend service resource.
+
+  Recreating a server TLS policy that's in use by another resource will give a `resourceInUseByAnotherResource` error. Use `lifecycle.create_before_destroy` to avoid this type of error.
```

Comment:
```suggestion
```

not needed to be in the Tls policy documentation, put the details of this interaction in the HttpsProxy resourceref field. specifically the 

> "If you remove this field from your configuration at the same time as deleting or recreating a referenced ServerTlsPolicy resource you will receive a `resourceInUseByAnotherResource` error. Use `lifecycle.create_before_destroy` within the ServerTlsPolicy resource to avoid this."